src/components/card/index.tsx

```diff
@@ -1,13 +1,170 @@
 import * as React from 'react'
+import {createNamedContext} from '@reach/utils/context'
+import {useId} from '@reach/auto-id'
+import type * as Polymorphic from '@reach/utils/polymorphic'
+
+const CardContext = createNamedContext<InternalCardContextValue>(
+  'CardContext',
+  {} as InternalCardContextValue,
+)
+
+type InternalCardContextValue = {
+  cardId: string | undefined
+  quiet: boolean
+  horizontal: boolean
+}
+
+type CardProps = {
+  quiet?: boolean
+  horizontal?: boolean
+}
+
+const cardDefaultClasses = `bg-white dark:bg-gray-800 dark:text-gray-200 shadow-sm rounded-lg overflow-hidden sm:p-8 p-5`
+
+const Card = React.forwardRef(function Card(
+  {
+    children,
+    as: Comp = 'div',
+    quiet = false,
+    horizontal = false,
```

Comment:
I put `horizontal` in here thinking that it might be where we want to control the general layout, but it would require quite a bit of internal complexity and defining rules/styles to manage it so in the end, I decided that we could use these for semantic structure primarily and let the internal layout be mostly managed in composed structures like `HorizontalResourceCard`

left it in for discussion

Reply:
Totally agree with such approach when we have separate wrapper for each type of card with ability to manage card's subcomponents instead of complicating Card component itself.